feat: noto sans eu (#3647)
db-ui/mono: packages/foundations/assets/fonts/README.md

# Generate fonts

To generate optimal fonts we use [fonttools](https://github.com/fonttools/fonttools). To use the tools you need python installed:

> 1. Install [python](https://www.python.org/downloads/)

Reviewer: ```suggestion
1. Install [python](https://docs.python-guide.org/starting/installation/#installation)
```

I'm thinking about providing a link to a more generic description by not directing the users to (even though official) download links, but a description, that would even also mention "the right way"™️ by using homebrew, etc.

2. Install fonttools: `pip3 install fonttools`
3. Install brotli: `pip3 install brotli`

Afterward, you can generate a font by running the following command:

```shell

> pyftsubset NotoSans-Regular.ttf --layout-features=* --flavor=woff2 --unicodes-file=unicode-eu.txt --output-file=NotoSans-EU-Regular.woff2

Reviewer: ```suggestion
pyftsubset NotoSans-Regular.ttf --layout-features=* --flavor=woff2 --unicodes-file=unicode-eu.txt --output-file=NotoSans-Regular-EU.woff2
```

```

## Local development

You can use `packages/foundations/assets/fonts/generate-eu-fonts.ts` to generate all fonts inside this directory. To do so, run the following command:

```shell
node packages/foundations/assets/fonts/generate-eu-fonts.ts
```

> To check if everything works fine you can check all glyphs [here](https://opentype.js.org/glyph-inspector.html).

Reviewer: ```suggestion
To check if everything works fine you can check all glyphs with an [opentype.js.org glyph inspector tool](https://opentype.js.org/glyph-inspector.html).
```
